Author: Danny Milosavljevic <dannym@scratchpost.org>
The socket name ended up too long inside the container.
Use a shorter one.
--- a/pkg/authorization/authz_unix_test.go	2019-01-10 01:55:02.997985947 +0100
+++ b/pkg/authorization/authz_unix_test.go	2019-01-10 02:03:21.177439757 +0100
@@ -24,7 +24,7 @@
 )
 
 const (
-	pluginAddress = "authz-test-plugin.sock"
+	pluginAddress = "/tmp/authz-test-plugin.sock"
 )
 
 func TestAuthZRequestPluginError(t *testing.T) {
@@ -263,12 +263,7 @@
 
 // createTestPlugin creates a new sample authorization plugin
 func createTestPlugin(t *testing.T) *authorizationPlugin {
-	pwd, err := os.Getwd()
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	client, err := plugins.NewClient("unix:///"+path.Join(pwd, pluginAddress), &tlsconfig.Options{InsecureSkipVerify: true})
+	client, err := plugins.NewClient("unix:///"+path.Join("/", pluginAddress), &tlsconfig.Options{InsecureSkipVerify: true})
 	if err != nil {
 		t.Fatalf("Failed to create client %v", err)
 	}
